Added the blog of ApacheCon Asia Integration track (#600)
apache/camel-website: content/blog/2021/07/ApacheCon-Asia-2021/index.md

---
title: "ApacheCon Asia Integration Track schedules"

> date: 2021-07-10

Reviewer: ```suggestion
date: 2021-07-15
```

Let's get this published today

For those of you who don't know much about [Apache Camel](/) yet, I suggest you check out [Claus](https://twitter.com/davsclaus) and [Andrea](https://twitter.com/oscerd2)'s presentation on [Apache Camel 3: The next generation of enterprise integration](https://apachecon.com/acasia2021/zh/sessions/1071.html), [Claus](https://twitter.com/davsclaus) and Andrea](https://twitter.com/oscerd2) are the lead developers on Apache Camel and [Claus](https://twitter.com/davsclaus) is the author of [Camel in Action](https://www.manning.com/books/camel-in-action-second-edition?). They led the development of Apache Camel 3, so in this talk they will present many of the latest developments on Camel 3, as well as the new members of the Camel family:

* [Camel K](/camel-k/latest/) is a serverless integration platform that can be deployed on top of K8S, enabling low-code/no-code functionality and the ability to quickly stitch together large numbers of Camel connectors with the power of integration patterns.

> * [Camel Quarkus](/camel-quarkus/latest/) organically integrates [Apache Camel](/manual/latest/index.html) integration capabilities with its [component library](/components/3.10.x/index.html) and [Quarkus](https://quarkus.io/)  runtime, so that we can enjoy the performance gains of application loading, [developer fun](https://quarkus.io/vision/developer-joy) and [container-first](https://quarkus.io/vision/container-first) benefits that [Quarkus](https://quarkus.io/) provides while doing application integration development.

Reviewer: ```suggestion
* [Camel Quarkus](/camel-quarkus/latest/) organically integrates [Apache Camel](/manual/latest/index.html) integration capabilities with its [component library](/components/latest/index.html) and [Quarkus](https://quarkus.io/)  runtime, so that we can enjoy the performance gains of application loading, [developer fun](https://quarkus.io/vision/developer-joy) and [container-first](https://quarkus.io/vision/container-first) benefits that [Quarkus](https://quarkus.io/) provides while doing application integration development.
```

With the popularity of event-driven architectures in cloud-based web services, services are increasingly being integrated through event response, and [Nicola Ferraro](https://twitter.com/ni_ferraro) has recently implemented a very handy set of [Kamelets](/camel-k/latest/kamelets/kamelets.html) on top of [Camel K](/camel-k/latest/). With Kamelets, developers can leverage the 300+ components provided by Apache Camel to Use yaml to define their own cloud-based application integration scripts and deploy them to k8s with the tools provided by [Camel K](/camel-k/latest/). For further details on using Kamelets, follow [Nicola Ferraro](https://twitter.com/ni_ferraro)'s talk [From Camel to Kamelets: new connectors for event-driven applications](https://apachecon.com/acasia2021/sessions/1073.html).

> [Tadayoshi Sato](https://twitter.com/tadayosi) has integrated Camel Route into [Jupyter](https://jupyter.org/) with the help of [JBang](https://jbang.dev) so that we can make the intergration easier like scripting. For those who would like to see the implementation in detail, you can follow [Tadayoshi Sato](https://twitter.com/tadayosi)'s talk [Integration with Scripting - Easy Integration with Apache Camel and JBang](https://apachecon.com/acasia2021/sessions/1074.html).

Reviewer: ```suggestion
[Tadayoshi Sato](https://twitter.com/tadayosi) proposes the idea of integration scripting by combining Apache Camel with [JBang](https://jbang.dev). He demonstrates how easily we can write an intergration with the combination. For those who would like to know more about the idea, you can follow [Tadayoshi Sato](https://twitter.com/tadayosi)'s talk [Integration as a Script - Integration made easy with Apache Camel and JBang](https://apachecon.com/acasia2021/sessions/1074.html).
```

Based on suggestion by @tadayosi

> [Tadayoshi Sato](https://twitter.com/tadayosi) has integrated Camel Route into [Jupyter](https://jupyter.org/) so that we can show beginners the [Jupyter](https://jupyter.org/) interactive documentation of the The Camel Route example can be shown to beginners through the [Jupyter](https://jupyter.org/) interactive documentation and can be used for hands-on practice based on interactive web pages. For those who would like to see the implementation in detail, you can follow [Tadayoshi Sato](https://twitter.com/tadayosi)'s talk [Integration with Scripting - Easy Integration with Apache Camel and JBang](https://apachecon.com/acasia2021/sessions/1074.html).

Reviewer 2: Thanks for the introduction. However, this time I don't talk about my ICamel jupyter kernel project. Rather I'll talk about the idea of using Camel with JBang https://www.jbang.dev/ to make integrations easier like scripting.

Reviewer: @WillemJiang please take note of this.

@tadayosi can you suggest a different paragraph here?

Author: Sorry, I just saw the comments.  @tadayosi Please feel free to update those content.

Reviewer 2: Thanks, I saw your update but it still refers to Jupyter (which is not relevant to my session). How about this text?
> [Tadayoshi Sato](https://twitter.com/tadayosi) proposes the idea of integration scripting by combining Apache Camel with [JBang](https://jbang.dev). He demonstrates how easily we can write an intergration with the combination. For those who would like to know more about the idea, you can follow [Tadayoshi Sato](https://twitter.com/tadayosi)'s talk [Integration as a Script - Integration made easy with Apache Camel and JBang](https://apachecon.com/acasia2021/sessions/1074.html).

> In addition to the cool features of [Apache Camel](https://camel.apache.org/) described above, we have invited other speakers to share [how Huawei Cloud ROMA uses Apache Camel](https://apachecon.com/acasia2021/sessions/1075.html), [Apache Project ARM build](https://apachecon.com/acasia2021/sessions/1086.html), and topics related to [Apache Open Meeting integration](https://apachecon.com/acasia2021/sessions/1075.html). You can visit the [Integration track page](https://apachecon.com/acasia2021/tracks/integration.html) for more information on these sessions.

Reviewer: ```suggestion
In addition to the cool features of [Apache Camel](/) described above, we have invited other speakers to share [how Huawei Cloud ROMA uses Apache Camel](https://apachecon.com/acasia2021/sessions/1075.html), [Apache Project ARM build](https://apachecon.com/acasia2021/sessions/1086.html), and topics related to [Apache Open Meeting integration](https://apachecon.com/acasia2021/sessions/1075.html). You can visit the [Integration track page](https://apachecon.com/acasia2021/tracks/integration.html) for more information on these sessions.
```
